unyt/_physical_ratios.py

```diff
@@ -160,3 +160,7 @@
 pascal_per_atm = 101325.0
 m_per_inch = 0.0254
 m_per_ft = 0.3048
+
+# logarithmic units
+# IEC 60027-3
```

Comment:
Can you elaborate in this comment more what this is referring to? (e.g. with a link)

Reply:
done.